frontend/src/components/BookingCard.tsx

```diff
@@ -97,39 +83,46 @@ export default function BookingCard() {
       console.log(err)
     }
   }
-
+  // To change date month and year in <DateComponent> using unixtocalender function
   return (
     <>
-      {facilityBookings?.map((event) => (
-        <BookingContainer
-          key={event.bookingID}
-          onClick={() => history.push(PATHS.VIEW_FACILITY_BOOKING_ID + event.bookingID)}
-        >
-          <BookingLeftDisplay>
-            <DateComponent>
-              {days[ViewStartDate.getDay()]}{' '}
-              {ViewStartDate.getDate() + ' ' + months[ViewStartDate.getMonth()] + ' ' + ViewStartDate.getFullYear()}
-            </DateComponent>
-            <TimeComponent>
-              {get24Hourtime(event.startTime)} to {get24Hourtime(event.endTime)}
-            </TimeComponent>
-            <BookingCCAName>{event.ccaName ? event.ccaName : 'Personal'}</BookingCCAName>
-          </BookingLeftDisplay>
-          <BookingRightDisplay>
-            {event.userID === localStorage.getItem('userID') ? (
-              <Icon src={adminIcon} />
-            ) : (
-              <Icon
-                onClick={() => {
-                  fetchTelegram(event)
-                }}
-                src={messageIcon}
-              />
-            )}
-          </BookingRightDisplay>
-        </BookingContainer>
-      ))}
-      {facilityBookings.length === 0 && <p style={{ padding: '23px' }}>There are no bookings on this day!</p>}
+      {bookings?.length ? (
+        bookings.map((booking) => (
+          <BookingContainer
+            key={booking.bookingID}
+            onClick={() => history.push(PATHS.VIEW_FACILITY_BOOKING_ID + booking.bookingID)}
+          >
+            <BookingLeftDisplay>
+              <DateComponent>
+                {days[new Date(booking.startTime * 1000).getDay()]}{' '}
+                {new Date(booking.startTime * 1000).getDate() +
+                  ' ' +
+                  months[new Date(booking.startTime * 1000).getMonth()] +
+                  ' ' +
+                  new Date(booking.startTime * 1000).getFullYear()}
+              </DateComponent>
+              <TimeComponent>
+                {get24Hourtime(booking.startTime)} to {get24Hourtime(booking.endTime)}
+              </TimeComponent>
+              <BookingCCAName>{booking.ccaName ? booking.ccaName : 'Personal'}</BookingCCAName>
+            </BookingLeftDisplay>
+            <BookingRightDisplay>
+              {booking.userID === localStorage.getItem('userID') ? (
+                <Icon src={adminIcon} />
+              ) : (
+                <Icon
+                  onClick={() => {
+                    fetchTelegram(booking)
+                  }}
+                  src={messageIcon}
+                />
+              )}
+            </BookingRightDisplay>
+          </BookingContainer>
+        ))
+      ) : (
+        <p style={{ padding: '23px' }}>There are no bookings on this day!</p>
```

Comment:
can help to make this a styled component instead of using inline styling?

Reply:
Resolved in commit e718c10d9d3b1542bb775b42be52c24cee642352.